interop/hobbits/README.md

```diff
@@ -0,0 +1,71 @@
+# Initial Interop
+
+For the initial client interop, clients will utilize a simplified networking implementation and wire protocol. For more information please review the hobbits specification [here](https://github.com/deltap2p/hobbits).
+
+### Prereqs
+
+- targeting the frozen spec (v0.8)
+- all clients must pass reference tests
+- all clients must pass [hobbits integration tests](./hobbits-integration-tests.md)
+
+- all wire protocol implementations must pass [hobbits conformance tests](https://github.com/jrhea/eth2.0-pm/blob/patch-2/interop/hobbits-conformance-tests.md)
+
+## Stage One: Coordinated Start
+
+Clients simulate a coordinated genesis event and gossip blocks/attestations for each slot.  This stage demonstrates the ability of clients to finalize the chain starting from `Eth2Genesis`.
+
+### Prereqs
+
+- Ability to read in the N number of deposits required to trigger the `Eth2Genesis` log
+- Ability to gossip blocks
+- Static peering
+- TCP transport
+
+### Procedure
+
+For the initial round: 
```

Comment:
We should include the [constant config](https://github.com/ethereum/eth2.0-specs/tree/master/configs/constant_presets) to be used here. I recommend either "minimal" or something similarly small. We can add an even more minimal one to that set if we think valuable, but minimal should work well